File: Frontend/src/app/people/person/evaluation/evaluation.ts
import { BaseEntity } from '../../../classes/base-entity';

export class Evaluation extends BaseEntity {
  public personId: string;
  public roleId: string;
  public evaluator: string;
  public date: Date;
  public notes: string;
  public score: number;
  public total: number;
  public result: EvaluationResult;
}

export class EvaluationWithNames extends Evaluation {
  public jobTitle: string;
}

export enum EvaluationResult {
  Poor = 'Poor',
  Good = 'Good',
  Excelent = 'Excelent'
}

File: Backend/Entities/Evaluation.cs
using System;
using LinqToDB;
using LinqToDB.Mapping;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Backend.Entities
{
    public class Evaluation : BaseEntity
    {
        public Guid PersonId { get; set; }
        public Guid RoleId { get; set; }
        public Guid Evaluator { get; set; }
        public DateTime Date { get; set; }
        public string Notes { get; set; }
        public decimal Score { get; set; }
        public decimal Total { get; set; }
        [Column(DataType = DataType.VarChar)]
        public EvaluationResult Result { get; set; }
    }

    public class EvaluationWithNames : Evaluation
    {
        public string JobTitle { get; set; }
        public string EvaluatorPreferredName { get; set; }
        public string EvaluatorLastName { get; set; }
    }

    [JsonConverter(typeof(StringEnumConverter))]
    public enum EvaluationResult
    {
        [MapValue(nameof(Poor))]
        Poor,
        [MapValue(nameof(Good))]
        Good,
        [MapValue("Excelent")]
        [MapValue(nameof(Excellent))]
        Excellent
    }
}